app/javascript/mastodon/main.js

```diff
@@ -2,7 +2,7 @@ const perf = require('./performance');
 
 // import default stylesheet with variables
 require('font-awesome/css/font-awesome.css');
-require('../styles/application.scss');
+require('application-style');
```

Comment:
I would maybe name this `mastodon-application-style` so that it's clear this isn't a package on npm. :)